doc/01-user-guide.adoc

```diff
@@ -569,7 +569,41 @@ Useful when you need to select a third row from a table, for example.
 * Any non-special key represents an attribute and its value.
 * `:fn/` is a prefix followed by a supported query function.
 
-Examples:
+There are several query functions of the form `:fn/*`. Each query
```

Comment:
Would you be so kind as to follow the [one sentence per line adoc](https://github.com/clj-commons/etaoin/blob/master/doc/02-developer-guide.adoc#docs) convention?
I find it makes future diffing easier.

Reply:
Yep, will do.